client/components/comment.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-layout row>
+    <v-flex xs2 class="text-xs-center">
+      <v-avatar size="36px"><img :src="comment.author.avatarUrl()" :alt="comment.author.username" :title="comment.author.username"></v-avatar>
+    </v-flex>
+    <v-flex xs8>
+      <div>
+        <div class="comment__body">{{comment.body}}</div>
+        <transition name="comment__details">
+          <div v-show="comment.showDetails">
```

Comment:
When a thread gets collapsed, we should probably also hide all details. Now if you have details shown, thread gets collapsed, you expand it, details show again.